pkg/controller/zookeepercluster/zookeepercluster_controller.go

```diff
@@ -467,7 +467,7 @@ func (r *ReconcileZookeeperCluster) yamlConfigMap(instance *zookeeperv1beta1.Zoo
 }
 
 func (r *ReconcileZookeeperCluster) reconcileFinalizers(instance *zookeeperv1beta1.ZookeeperCluster) (err error) {
```

Comment:
Looks like we need coverage of `reconcileFinalizers`, but it isn't currently there, so I'll file that in a separate issue.